ui/pages/password-reset/index.js

```diff
@@ -13,6 +13,7 @@ export default function PasswordReset() {
   const [submitted, setSubmitted] = useState(false)
 
   async function onSubmit(e) {
+    setSubmitted(true)
```

Comment:
You may want to set this back to false on error so the user can try again.

Reply:
Good call out though but the UI doesn't currently branch on errors. The error state is never set to a non-empty value so I've removed it